spec/index.html

```diff
@@ -493,12 +493,8 @@ <h3>IRIs</h3>
       comparing IRIs for equality.</p>
 
       <p class="issue" data-number="15">
-        A future version of this specification may use [[?URL]],
-        as [[?RFC3987]] is an abandoned specification.
-        This would include retaining most of the `IRI`-based nomenclature, but
-        defining in terms of
-        <a data-cite="URL#absolute-url-with-fragment-string">absolute-URL with fragment string</a>
-        and <a data-cite="?URL#relative-url-string">relative-URL string</a>.
+        The group is aware that work has not progressed on [[RFC3987]],
+        and is considering alternatives.
```

Comment:
Given Martin's comments `<p>` can be removed.

#15 is quite limied -- "Revise terminology to align with current RFCs" from [PA's observation of incorrect use of "relative IRI"](https://lists.w3.org/Archives/Public/public-rdf-comments/2021Mar/0002.html). 

If you want something :

```suggestion
        The group is updating IRI terminology to more closer follow the current RFCs.
```